fbw-common/src/systems/shared/src/FbwAircraftSentryClient.ts

```diff
@@ -5,9 +5,6 @@
 /* eslint-disable no-console */
 
 import * as Sentry from '@sentry/browser';
-import { BrowserTracing } from '@sentry/tracing';
-import { CaptureConsole as CaptureConsoleIntegration } from '@sentry/integrations';
-import { Integration } from '@sentry/types';
```

Comment:
All of these integrations are now directly part of `@sentry/browser` so we can remove a couple of direct dependencies from the repo :)